azavea/iow-boundary-tool
# Add test boundary and submissions

## Overview
Add three mock submissions (a draft, a submission, and an approved submission) in order to continue mocking the app. For development only.

Closes #106

### Demo
<img width="630" alt="Screen Shot 2022-10-06 at 4 42 07 PM" src="https://user-images.githubusercontent.com/38668450/194414569-40cd07e8-8697-413a-9e1b-96e22c6eab3b.png">

## Testing Instructions

- scripts/resetdb
- Use the Django admin to view the test submissions, verify acceptance criteria in #106

The Django admin does not have all relevant fields enabled, so to be super thorough, you could open `scripts/manage shell` and do something like:

<details>

```python
>>> from pprint import pprint
>>> from api.models.submission import Submission
>>> for s in Submission.objects.all():
...   pprint(vars(s))
... 
{'_state': <django.db.models.base.ModelState object at 0xffff99882fd0>,
 'boundary_id': 1,
 'created_at': datetime.datetime(2022, 10, 6, 20, 37, 19, 829320, tzinfo=<UTC>),
 'created_by_id': 3,
 'id': 1,
 'notes': '',
 'shape': <Polygon object at 0xffff9987c390>,
 'submitted_at': None,
 'submitted_by_id': None,
 'updated_at': datetime.datetime(2022, 10, 6, 20, 37, 19, 829333, tzinfo=<UTC>),
 'upload_edited_at': None,
 'upload_filename': ''}
{'_state': <django.db.models.base.ModelState object at 0xffff99882a00>,
 'boundary_id': 1,
 'created_at': datetime.datetime(2022, 10, 6, 20, 37, 19, 832722, tzinfo=<UTC>),
 'created_by_id': 3,
 'id': 2,
 'notes': 'Notes for the test submission.',
 'shape': <Polygon object at 0xffff9987c710>,
 'submitted_at': datetime.datetime(2022, 10, 2, 12, 0, tzinfo=<UTC>),
 'submitted_by_id': 3,
 'updated_at': datetime.datetime(2022, 10, 6, 20, 37, 19, 832736, tzinfo=<UTC>),
 'upload_edited_at': None,
 'upload_filename': ''}
{'_state': <django.db.models.base.ModelState object at 0xffff995670a0>,
 'boundary_id': 1,
 'created_at': datetime.datetime(2022, 10, 6, 20, 37, 19, 835451, tzinfo=<UTC>),
 'created_by_id': 3,
 'id': 3,
 'notes': 'Notes for the test approved boundary.',
 'shape': <Polygon object at 0xffff9987c410>,
 'submitted_at': datetime.datetime(2022, 10, 1, 9, 0, tzinfo=<UTC>),
 'submitted_by_id': 3,
 'updated_at': datetime.datetime(2022, 10, 6, 20, 37, 19, 835465, tzinfo=<UTC>),
 'upload_edited_at': None,
 'upload_filename': ''}
```
</details>

## Checklist
